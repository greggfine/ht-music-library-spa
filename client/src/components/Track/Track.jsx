import React from "react";
import "./Track.styles.scss";
import ListGroup from "react-bootstrap/ListGroup";

export default function Track({ filename }) {
  return (
    <ListGroup.Item className="Track" action>
      {filename}
      {/* <audio src={`audio/${filename}`} controls></audio> */}
    </ListGroup.Item>
  );
}
