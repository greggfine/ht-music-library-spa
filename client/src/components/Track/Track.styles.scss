.Track {
  display: flex;
  justify-content: space-around;
}
