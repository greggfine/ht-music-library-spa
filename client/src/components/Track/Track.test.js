import React from "react";
import Track from "./Track";
import { render, cleanup } from "@testing-library/react";

afterEach(cleanup);

describe("<Track>", () => {
  it("renders", () => {
    const { asFragment } = render(<Track />);
    expect(asFragment()).toMatchSnapshot();
  });
});
