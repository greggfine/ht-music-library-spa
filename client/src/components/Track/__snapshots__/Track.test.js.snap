// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Track> renders 1`] = `
<DocumentFragment>
  <li
    class="MuiListItem-root MuiListItem-gutters"
  >
    <audio
      controls=""
      src="audio/undefined"
    />
  </li>
</DocumentFragment>
`;
